### PR TITLE
Add color scheme support

### PR DESCRIPTION
**Pull request by rockerBOO**
### Goal:
To have a darker version of the site.

### Features:  
* Color scheme support. 
* Uses CSS variables to modify the colors. 
* Default color scheme is the light version. 
* Uses CSS Grid.
* Add Calibri font to default font list.

### Screens:
https://imgur.com/a/JX0AyXb

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 
-    <div class="col-narrow">
-      This site's source is <a href="https://github.com/neovim/neovim.github.io/">hosted on GitHub</a>.
-    </div>
-  </div>
+		<div>
+			This site's source is
+			<a href="https://github.com/neovim/neovim.github.io/">hosted on GitHub</a
+			>.
+		</div>
```
Comment (justinmk) [inline _includes/footer.html:13]: please undo all cosmetic formatting changes so that the PR is easier to review

**Comment by justinmk** [comment]
Thanks, it's great to have help with the website! If this is a small change and doesn't bloat the site size too much, I don't have an objection.

Can you show a screenshot of the docs https://neovim.io/doc/user/ ? Most of the rest of the site is not something users are likely to spend more than 5 minutes looking at, so I don't see the purpose of theming. Not to mention, the website HTML is pretty "clean" so it should be trivial for users to theme the site using a browser extension or something.

**Comment by rockerBOO** [comment]
Thank you Justin!

I couldn't figure out how to get `/doc/` to generate. Is there some documentation for how to do that? 

My main goal was I am learning about the new LSP integration, and I am sharing particular pages with others who are also learning it. I spend maybe hours looking at the page currently. The lighter page can be tough to work with and I would rather not have to install a plugin as most websites I use have a dark version. 

My vim experience is low, and also maybe realizing late, that I could use the documentation inside Neovim. 

---

* Didn't mean to change the formatting so I will fix that. 
* Once I know how to generate the docs pages, I will do that and clean up any inconsistencies with the styling.

**Comment by justinmk** [comment]
> I couldn't figure out how to get `/doc/` to generate. Is there some documentation for how to do that?

You could edit the online version of https://neovim.io/doc/user/ in chrome/firefox devtools , don't need to generate the docs.

**Comment by rockerBOO** [comment]
User manual
Mobile

Seems to look good to me. 

Thanks Justin!

**Comment by justinmk** [comment]
yeah looks pretty good. so the formatting needs to be fixed before I can actually review this...

**Comment by rockerBOO** [comment]
Where is my formatting bad? I had to make changes on the modified files (removed the unmodified formatted files). Do I need to undo more formatting for those files?

**Comment by justinmk** [comment]
Ok these changes actually look pretty thoughtful. Nice work.

**Comment by justinmk** [comment]
how does one actually activate the dark mode?

**Comment by rockerBOO** [comment]
Windows has an option to set your preference to dark. 
Firefox has a `about:config` to set a preference (`ui.systemUsesDarkTheme = 1`).
GTK has `gtk-application-prefer-dark-theme=1`

@justinmk thank you! Thank you for your help!

**Comment by cweagans** [comment]
Should we think about an alternate version of the logo that uses light text for dark mode? Grey on grey is a bit hard to read.

**Pull request merged**